### PR TITLE
Added dashboard sketch, updates to README and PROPOSAL 

### LINKED ISSUE #2: Proposal section 2: description of the data

### LINKED ISSUE #4: Sketch the dashboard

### PR DESCRIPTION
**Pull request by iamMoid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,37 @@
...
+
 ## Dashboard description
 
-Our dashboard consists of one web page that shows 4 main reactive plots:
+Our dashboard consists of one web page that shows overall summary and 4 main reactive plots:
+
+- The high level totals
 
-- The top rated movies by genre
+These are a set of 4 numeric totals showing the total count of movies, the average runtime across all movies, the total number of actors across all movies, and the average rating of all movies based on the filters applied.
```
Comment (ming0701) [inline README.md:11]: - if the first chart is "the proportion of movies by genre" we should change it back to bar chart

File: README.md [hunk 2]
```diff
@@ -1,29 +1,37 @@
...
+This is a box plot showing the genres on the x-axis and the count of movies on the y-axis. The box plots will be categorized by a categorical column, for e.g., top 3 regions.
 
 - How movie ratings vary over time
 
 This is a line chart showing the years on the x-axis and the mean rating per genre on the y-axis. The individual lines may be coloured by other features such as genre or region of production.
 
 - The top rated movies by actor
 
-This is a bubble graph showing the mean rating on the y-axis and the year on the x-axis. Each bubble represents an actor and the size of the bubble represents the number of movies that that actor performed in. The user can choose the number of actors that are depicted for each year using a slider.
+This is a bubble graph where each bubble represents an actor and the size and color of the bubble represents the number of movies that that actor performed in. The user can choose the number of actors that are depicted for each year using a slider.
```
Comment (ming0701) [inline README.md]:
```suggestion
This is a bubble graph where each bubble represents an actor and the size and color of the bubble represents the number of movies that the actor performed in. The user can choose the number of actors that are depicted for each year using a slider.
```

**Review by ming0701 — COMMENTED** [review]
minor comments above

**Review by ming0701 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (iamMoid) [reply README.md:11]: As discussed, updated to "The distribution of movies by genre"

File: README.md (hunk 2 above)
Comment (iamMoid) [reply README.md]: Updated the description for bubble chart

**Pull request merged**